### ISSUE TITLE
Dialogs may be a different size if opened while zoomed in

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/sun#611]
For https://github.com/phetsims/QA/issues/520
When you open the about dialog while zoomed in to the sim, it seems to be larger (in comparison to the rest of the sim, not just the obvious) than if it is opened when not zoomed in. It does not go off screen from what I can see, is very minor, and I believe the dialog code has already changed since this SHAs so I don't think anything really needs to be done here, but feel free to disagree.

Steps:
1. Put iPad in landscape mode
2. Zoom in to sim.
3. Open about dialog
4. Zoom out and observe size
5. Close out of about dialog
6. Open about dialog again while fully zoomed out. Observe size of dialog

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/41024075/89071716-8026ad80-d334-11ea-8941-1dcde9c386f5.gif)

**Comment by jessegreenberg** [comment]
I am having a difficult time reproducing this. I haven't been able to see this yet on my Windows or iPad devices. Over slack @KatieWoe mentioned that it only seemed to happen in landscape mode, but I haven't seen it in landscape or portrait mode yet.

**Comment by jessegreenberg** [comment]
I can produce the bug with `stringTest=double`. So the dialog needs to be larger for this to happen.

**Comment by jessegreenberg** [comment]
OK, I verified that this is not an issue in master because of recent changes to Dialog that position pop ups based on layoutBounds.

**Comment by jessegreenberg** [comment]
I spent some time trying to figure out why it was happening on the old shas. I was able to verify that `bounds`, `screenBounds`, and `scale` used by dialog layout are not changing with zoom which is as I would expect. I also made sure that even longer strings like stringTest=long didn't cause worse problems with layout in the MAL RC.

Since this is fixed on master I am going to go ahead and close, I don't think it is worth getting to the bottom of this since we wouldn't fix in branch anyway.

**Comment by jessegreenberg** [comment]
Actually, reopening for @KatieWoe to verify that this behavior is indeed fixed in master.

**Comment by KatieWoe** [comment]
This does look fixed on master
